### PR TITLE
Fix the wrong Postgres SQL of nested include when quoteIdentifiers is false

### LINKED ISSUE #6351: Nested include causes wrong sql on postgres when quoteIdentifiers is false
## What you are doing?
_Post a minimal code sample that reproduces the issue, including models and associations_

```js
test('nested include (left outer join)', function () {
      var User = Support.sequelize.define('User', {
          name: DataTypes.STRING,
          age: DataTypes.INTEGER
        },
        {
          freezeTableName: true
        });
      var Post = Support.sequelize.define('Post', {
          title: DataTypes.STRING
        },
        {
          freezeTableName: true
        });
      var Comment = Support.sequelize.define('Comment', {
          title: DataTypes.STRING
        },
        {
          freezeTableName: true
        });

      User.Posts = User.hasMany(Post, {foreignKey: 'user_id'});
      Post.Comments = Post.hasMany(Comment, {foreignKey: 'post_id'});

      expectsql(sql.selectQuery('User', {
        attributes: ['name', 'age'],
        include: Model.$validateIncludedElements({
          include: [{
            attributes: ['title'],
            association: User.Posts,
            include: [
              {
                model: Comment
              }
            ]
          }],
          model: User
        }).include,
        model: User
      }, User), {
        default: 'SELECT [User].[name], [User].[age], [Posts].[id] AS [Posts.id], [Posts].[title] AS [Posts.title], [Posts.Comments].[id] AS [Posts.Comments.id], [Posts.Comments].[title] AS [Posts.Comments.title], [Posts.Comments].[createdAt] AS [Posts.Comments.createdAt], [Posts.Comments].[updatedAt] AS [Posts.Comments.updatedAt], [Posts.Comments].[post_id] AS [Posts.Comments.post_id] FROM [User] AS [User] LEFT OUTER JOIN [Post] AS [Posts] ON [User].[id] = [Posts].[user_id] LEFT OUTER JOIN [Comment] AS [Posts.Comments] ON [Posts].[id] = [Posts.Comments].[post_id];',
        postgres: 'SELECT User.name, User.age, Posts.id AS "Posts.id", Posts.title AS "Posts.title", "Posts.Comments".id AS "Posts.Comments.id", "Posts.Comments".title AS "Posts.Comments.title", "Posts.Comments".createdAt AS "Posts.Comments.createdAt", "Posts.Comments".updatedAt AS "Posts.Comments.updatedAt", "Posts.Comments".post_id AS "Posts.Comments.post_id" FROM User AS User LEFT OUTER JOIN Post AS Posts ON User.id = Posts.user_id LEFT OUTER JOIN Comment AS "Posts.Comments" ON Posts.id = "Posts.Comments".post_id;'
      });
    });

  });
```

## What do you expect to happen?
The sql should generated as expecedt

## What is actually happening?
```sql
SELECT 
User.name, 
User.age, Posts.id AS "Posts.id", 
Posts.title AS "Posts.title", 
Posts.Comments.id AS "Posts.Comments.id", #wrong
Posts.Comments.title AS "Posts.Comments.title",  #wrong
Posts.Comments.createdAt AS "Posts.Comments.createdAt", #wrong
Posts.Comments.updatedAt AS "Posts.Comments.updatedAt", #wrong
Posts.Comments.post_id AS "Posts.Comments.post_id" #wrong
FROM User AS User 
  LEFT OUTER JOIN Post AS Posts 
    ON User.id = Posts.user_id 
  LEFT OUTER JOIN Comment AS Posts.Comments #wrong
    ON Posts.id = Posts.Comments.post_id; 
```
The nested table alias has no quote.

__Dialect:__  postgres 
__Database version:__ any
__Sequelize version:__ 3.23.6

### PR DESCRIPTION
**Pull request by topikachu**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Does your issue contain a link to existing issue (Closes #[issue]) or a description of the issue you are solving?  #6351
- [x] Have you added new tests to prevent regressions?
- [x] Have you added an entry under `Future` in the changelog?

### Description of change

Alwasy quote identifier which has '.'

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -30,6 +30,8 @@
 - [FIXED] `attribute:[]` throw errors with `include` or `through` [#5078](https://github.com/sequelize/sequelize/issues/5078) [#4222](https://github.com/sequelize/sequelize/issues/4222) [#5958](https://github.com/sequelize/sequelize/issues/5958) [#5590](https://github.com/sequelize/sequelize/issues/5590) [#6139](https://github.com/sequelize/sequelize/issues/6139) [#4866](https://github.com/sequelize/sequelize/issues/4866) [#6242](https://github.com/sequelize/sequelize/issues/6242)
 - [SECURITY] `GEOMETRY` and `GEOGRAPHY` SQL injection attacks [#6194](https://github.com/sequelize/sequelize/issues/6194)
 - [FIXED] `DECIMAL` now supports `UNSIGNED` / `ZEROFILL` (MySQL) [#2038](https://github.com/sequelize/sequelize/issues/2038)
+- [FIXED] Generate correct SQL of nested include when quoteIdentifiers is false. (Postgres) [#6351](https://github.com/sequelize/sequelize/issues/6351)
+
```
Comment (sushantdhiman) [inline changelog.md]: Remove this extra line

File: lib/dialects/postgres/query-generator.js [hunk 2]
```diff
@@ -746,6 +746,9 @@ const QueryGenerator = {
 
   quoteIdentifier(identifier, force) {
     if (identifier === '*') return identifier;
+    if (identifier.indexOf('.') !== -1){
```
Comment (sushantdhiman) [inline lib/dialects/postgres/query-generator.js]: space after condition like ` if (identifier.indexOf('.') !== -1) {`

Comment (BridgeAR) [inline lib/dialects/postgres/query-generator.js]: To only check this if needed I'd move the condition into the already existing if. 
As in:
```js
if (!force && this.options && this.options.quoteIdentifiers === false && identifier.indexOf('.') === -1) { // default is `true`
```

Comment (sushantdhiman) [inline lib/dialects/postgres/query-generator.js]: If the force var is not used in next code sections thats actually better

**Comment by topikachu** [comment]
@sushantdhiman @BridgeAR 
Would you please review the new PR again?
Thanks

**Comment by BridgeAR** [comment]
LGTM

**Pull request merged**